requirements.txt

```diff
@@ -1,23 +1,18 @@
-connexion[swagger-ui]==2.7.0
-email-validator
-fasteners
-flask==1.1.2
-flask-authorize
-flask-jwt-extended>=4.1.0
-flask_bootstrap==3.3.7.1
-flask_cors==3.0.8
-flask_debugtoolbar==0.11.0
-flask_login==0.5.0
-flask_mail==0.9.1
-flask_marshmallow==0.12.0
-flask_moment==0.9.0
-flask_sqlalchemy==2.4.1
-flask_wtf==0.14.3
-marshmallow==3.5.1
-marshmallow_sqlalchemy==0.23.0
-python_dateutil==2.8.1
-requests=2.23.0
-seamm_util
-selenium==3.141.0
-sqlalchemy==1.3.16
-wtforms==2.3.1
+connexion==2.7.0
+fasteners==0.16.3
+Flask==2.0.2
+Flask_Authorize==0.2.6
+Flask_Bootstrap==3.3.7.1
+Flask_Cors==3.0.10
+Flask_DebugToolbar==0.11.0
+Flask_JWT_Extended==4.3.1
+Flask_Mail==0.9.1
+flask_marshmallow==0.14.0
+Flask_Moment==1.0.2
+Flask_SQLAlchemy==2.5.1
+Flask_WTF==0.15.1
+seamm_datastore==2021.11.21
+seamm_util==2021.8.26
+six==1.16.0
+SQLAlchemy==1.4.26
+WTForms==2.3.3
```

Comment:
Should we consider loosening the pinning here, or is it something that's required for releasing?